src/main/java/stream/flarebot/flarebot/util/GeneralUtils.java

```diff
@@ -643,4 +644,36 @@ public static String getVerificationString(Guild.VerificationLevel level) {
                 return level.toString().charAt(0) + level.toString().substring(1).toLowerCase();
         }
     }
+
+    public static Long parseTime(String time) {
+        List<String> toReverse = Arrays.asList(time.split(":"));
+        Collections.reverse(toReverse);
+        String timeToUse = toReverse.stream().collect(Collectors.joining(":"));
+        Matcher digitMatcher = Pattern.compile("^([0-9]*):?([0-9]*)?:?([0-9]*)?$").matcher(timeToUse);
```

Comment:
Make this a constant so it is only compiled once and then re-used. Doing this severely slows down parsing as Regex is slow.

Reply:
:+1: